drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-client/src/main/java/org/drools/workbench/screens/testscenario/client/reporting/TestRunnerReportingScreen.java

```diff
@@ -44,45 +50,79 @@ public TestRunnerReportingScreen() {
     }
 
     @Inject
-    public TestRunnerReportingScreen( TestRunnerReportingView view,
-                                      TestRuntimeReportingService testRuntimeReportingService ) {
+    public TestRunnerReportingScreen(TestRunnerReportingView view,
+                                     TestRuntimeReportingService testRuntimeReportingService) {
         this.view = view;
-        view.setPresenter( this );
-        view.bindDataGridToService( testRuntimeReportingService );
+        view.setPresenter(this);
     }
 
     @DefaultPosition
     public Position getDefaultPosition() {
-        return CompassPosition.SOUTH;
+        return CompassPosition.EAST;
     }
 
     @WorkbenchPartTitle
     public String getTitle() {
-        return "Reporting";
+        return "";
```

Comment:
I decided to go with the header from the frontend, so I deactivated the original header. But I realized now it is not exactly according the uc proposal (as the headline is not aligned with the asset toolbar), I can change it to be, even though I am afraid it will be uglier.

Reply:
I think we should return `"Test Report"` here and remove [1] from `TestRunnerReportingViewImpl.html`.

[1]
```html
<h3 class="kie-dock__heading">
                    Test Report
</h3>
<hr class="kie-dock__divider"/>
```

Reply:
Ok, I will change it.